### ISSUE TITLE
v1.20 my .vbs scripts not working

### ISSUE THREAD

**Issue by nr1995** [Genymobile/scrcpy#2778]
Hi, noob here.
These little .vbs scripts of mine worked until v1.20.
Now they open a new adb.exe in Task Manager but nothing appears.
Am I missing something?
Thanks

strCommand = "cmd /c scrcpy.exe --max-fps 1 --crop 260:70:415:93 --window-title WeChat --window-x 1792 --window-y 977 --window-width 88 --window-height 22 --window-borderless --always-on-top"
For Each Arg In WScript.Arguments
    strCommand = strCommand & " """ & replace(Arg, """", """""""""") & """"
Next
CreateObject("Wscript.Shell").Run strCommand, 0, false

**Comment by rom1v** [comment]
Replace the last line by:

```
CreateObject("Wscript.Shell").Run strCommand, 1, false
WScript.Sleep 5000
```

Then run `scrcpy-noconsole.vbs` from a console (`cmd.exe`), you'll have time to see the error messages.

**Comment by nr1995** [comment]
Oh, the problem's not with the script.
The problem is that I can only have one window open at a time.
Whichever script I open first is OK.  Whichever I try second won't work.
I only have one phone connected, btw.

**Comment by rom1v** [comment]
Did it work on v1.19?

What's the error for the second scrcpy instance?

**Comment by nr1995** [comment]
Yeah I'd been happily using several .vbs scripts at once on v1.19 for a while.

Sorry I'm not very technical, so I probably did it wrong,
but I didn't seem to get any error message in the console window.

I typed    cd /d D:\phone stuff\scrcpy   to get to the SCRPCY folder,
then typed    scrcpy-noconsole.vbs    to run that script, which worked,
then typed    battery.vbs    to open another one of my .vbs which worked in v1.19,
but nothing happened.  No error either.  Even though it had the changed last few lines like you gave me.

**Comment by rom1v** [comment]
Could you try without your vbs script.

Just open two scrcpy instances (both over USB, both other wifi, or one USB one wifi).

Refs #2779

**Comment by nr1995** [comment]
I tried two of scrcpy-noconsole.vbs (which I haven't edited), but just the same as before.
First one is fine, second one does nothing.
Everything is over USB.

**Comment by rom1v** [comment]
> The problem is that I can only have one window open at a time.

Please test #2783, it should fix this issue.

**Comment by rom1v** [comment]
@nr1995 I just pushed a"final" version (hopefully) of the fix in #2783. I provide new binaries on that PR.

Could you please test them and confirm everything works as expected for you?

I think I will publish a new hotfix release (1.20.1) for this bug soon.

**Comment by rom1v** [comment]
Fixed by #2783. If this is not the case, please reopen.

**Comment by rom1v** [comment]
Thank you for your feedback!
